### PR TITLE
More docker support

### PR DESCRIPTION
**Pull request by yangm97**

### CODE REVIEW

**Comment by yangm97** [comment]
Make sure to set `$DOCKER_NAMESPACE` to `yggdrasilnetwork/yggdrasil-go`before merging this.

File: .circleci/config.yml [hunk 1]
```diff
@@ -159,35 +160,97 @@ jobs:
...
+                cp /tmp/upload/${CINAME}-${CIVERSION}-yggdrasilctl-linux-armhf linux/arm/v6/yggdrasilctl
+              mkdir -p linux/arm/v7 && \
+                cp /tmp/upload/${CINAME}-${CIVERSION}-linux-armhf linux/arm/v7/yggdrasil
+                cp /tmp/upload/${CINAME}-${CIVERSION}-yggdrasilctl-linux-armhf linux/arm/v7/yggdrasilctl
+              mkdir -p linux/arm64/v8 && \
+                cp /tmp/upload/${CINAME}-${CIVERSION}-linux-arm64 linux/arm64/v8/yggdrasil
+                cp /tmp/upload/${CINAME}-${CIVERSION}-yggdrasilctl-linux-arm64 linux/arm64/v8/yggdrasilctl
+              docker login -u $DOCKER_USER -p $DOCKER_PASS
+              buildx build \
+                --push \
```
Comment (neilalexander) [inline .circleci/config.yml]: What are the prerequisites needed for pushing?

Comment (yangm97) [reply .circleci/config.yml]: On CircleCI variables:
- set `$DOCKER_NAMESPACE` to `yggdrasilnetwork/yggdrasil-go`
- set `$DOCKER_USER` to your (or a dummy) docker hub account user
- set `DOCKER_PASS` to the password of the account above

**Comment by neilalexander** [comment]
Closing as this is stale and I don't think anyone has ever particularly asked us about Docker anyway.

**Comment by 0x1a8510f2** [comment]
For the record, I'm using yggdrasil in Docker but have been using a custom build because https://hub.docker.com/r/yggdrasilnetwork/yggdrasil-go is out of date.

(If anyone else wants to use my builds, they are accessible at `dev.l1qu1d.net/archive/yggdrasil-go:latest`, built by https://ci.l1qu1d.net/archive/yggdrasil-go from https://dev.l1qu1d.net/archive/yggdrasil-go using the config at https://dev.l1qu1d.net/l1qu1d-net/ci/src/branch/master/yggdrasil-public-node-docker.drone.yml)

**Pull request closed without merging**